.github/workflows/bindings_ci.yml

```diff
@@ -164,14 +164,14 @@ jobs:
           # keep in sync with uniffi dependency in Cargo.toml's
           args: uniffi_bindgen --git https://github.com/mozilla/uniffi-rs --rev 091c3561656e72e1a4160412c83b36d98e556d06
 
-      - name: Generate .xcframework
+      - name: Build library & bindings
         working-directory: bindings/apple
-        run: sh ./debug_build_xcframework.sh x86_64 ci
```

Comment:
Can we remove the `debug_build_xcframework.sh` file then?

Reply:
Well, we currently use that script (without the arguments) for local development on EX, so not right now.

But I don't think its going to be long until we look into simply building a static lib as part of the Xcode build process tbh.